src/context/StorageContext.tsx

```diff
@@ -47,13 +49,20 @@ export function StorageContextProvider({ children }: PropsWithChildren): JSX.Ele
     });
   };
 
+  const reset = (): void => {
+    setData(() => {
+      sessionStorage.removeItem(STORAGE_KEY);
+      return { json: undefined, depositContractCalldata: undefined };
```

Comment:
to simply this returning an empty object will result same
```suggestion
      return {};
```